genie-web/src/test/resources/application-ci.yml

```diff
@@ -30,6 +30,10 @@ genie:
   health:
     maxCpuLoadPercent: 100
   jobs:
+    clusters:
```

Comment:
It is cause I don't want someone starting up Genie without knowledge of the `ScriptLoadBalancer` and seeing a bunch of Stack Traces periodically cause the system couldn't find the script. But for the integration test I wanted to make sure that the system could still run even if the `ScriptLoadBalancer` was enabled and not configured correctly. I could take it back out but it doesn't really hurt anything for the testing. Ideally this would pull a script to actually test running but the UnitTest already does that.